### ISSUE TITLE
Favicon displays

### ISSUE THREAD

**Issue by JoaoNM** [minbrowser/min#1855]
Would really like to have favicon displays for better UX so users can quickly find tabs they're looking for

**Comment by subodhpareek18** [comment]
Yes that's a good idea, but perhaps make it optional in the setting?

**Comment by fdemir** [comment]
Any updates on this? I would like to contribute if you give me some detail on it.

**Comment by sosise** [comment]
I also need this "feature"
